Makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: build
+build:
+	mkdir -p bin && export CGO_ENABLED=0 GOOS=linux && go build -o bin/app -ldflags '-s -w' ./core/main.go
+
+.PHONY: swagger-build
+swagger-build:
+ifeq ($(shell uname -m),arm64)
+	docker build -t horizon-swagger -f build/swagger/Dockerfile . --platform linux/arm64
+else
+	docker build -t horizon-swagger -f build/swagger/Dockerfile .
+endif
+
```

Comment:
Should add command for building job and cleaning.

Reply:
added